static/js/constants.js

```diff
@@ -231,5 +231,5 @@ export const DASHBOARD_RESPONSE = [
 
 export const DATE_FORMAT = 'YYYY-MM-DD';
 
-export const DASHBOARD_COURSE_HEIGHT = 70;
```

Comment:
Please make a separate constant. We don't necessarily want the two numbers to be the same.